README.md

```diff
@@ -1,23 +1,25 @@
-# InfluxData Helm Charts
+# InfluxData Helm charts
 
 [![License](https://img.shields.io/badge/license-MIT-green.svg)](./LICENSE)
 [![](https://github.com/influxdata/helm-charts/workflows/helm-charts%2Frelease/badge.svg?branch=master)](https://github.com/influxdata/helm-charts/actions)
 
-
-This functionality is in beta and is subject to change. The code is provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+InfluxData Helm charts are in beta and subject to change. The code is provided as-is with no warranties. Beta features are not covered by the [Software License Subscription Agreement (SLSA)](https://www.influxdata.com/legal/slsa/).
 
 ## Usage
 
-[Helm](https://helm.sh) must be installed to use the charts.
-Please refer to Helm's [documentation](https://helm.sh/docs/) to get started.
+1. Install [Helm](https://helm.sh). For more information, see [Helm documentation](https://helm.sh/docs/).
```

Comment:
Does it matter which version of Helm? Want to include the appropriate version as a prerequisite in other readmes.

Reply:
InfluxDB 2 chart is Helm 3 only, the rest should work on both.

Reply:
thanks, @rawkode - added as a prerequisite to all READMES